### ISSUE TITLE
Test running RStudio on Nimbus VMs

### ISSUE THREAD

**Issue by georgiesamaha** [Sydney-Informatics-Hub/rnaseqOct23_Testing#15]
Check RStudio can run on Nimbus VMs without issue. If any issues, create new ticket.

**Comment by nandan75** [comment]
- The RServer was run using the updated RStudio singularity containers as documented at https://github.com/Sydney-Informatics-Hub/rnaseqOct23_Testing/blob/Nandan/Scripts/Day2/Runtime-testing.md#run-the-singularity-container-to-start-the-rstudio-server

- The Rmarkdown file https://github.com/Sydney-Informatics-Hub/rnaseqOct23_Testing/blob/Nandan/Scripts/Day2/rnaseq_DE_analysis_Day2.Rmd was executed and it ran successfully. 

  - The updated R packages were tested for loading.
  - The R script which was modified at a few steps was tested 
  - In summary the R script for the downstream analysis with plot generation works fine

- The loop unavailable error pops up and can be solved locally using the workaround as document ed here - https://github.com/Sydney-Informatics-Hub/rnaseqOct23_Testing/blob/Nandan/Scripts/Day2/Runtime-testing.md#intermittent-problems-when-running-the-rserver
    - This needs to get solved for the instance in general.

**Comment by georgiesamaha** [comment]
## 2022 run 
For 2022 workshop we ran RStudio Singularity image from CVMFS 

Ran successfully with: 
```
PASSWORD='abc' singularity exec \
    -B /home/ubuntu/rnaseq23/day2_RStudio/working_dir/rstudio-server:/var/lib/rstudio-server \
    -B /home/ubuntu/rnaseq23/day2_RStudio/working_dir/rstudio-server:/var/run/rstudio-server \
    -B /home/ubuntu/rnaseq23/day2_RStudio/working_dir:/home \
    /cvmfs/containers.biocommons.aarnet.edu.au/r/n/rnaseq_rstudio.sif \
    rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user ubuntu
```
 
To quit we ran: 

``` 
CTRL + C
```

## 2023 run 

This year, when running an updated version of RStudio Singularity image (stored locally on VM)  

```
 PASSWORD='abc' singularity exec \
-B /home/ubuntu/rnaseq23/day2_RStudio/working_dir/rstudio-server:/var/lib/rstudio-server \
-B /home/ubuntu/rnaseq23/day2_RStudio/working_dir/rstudio-server:/var/run/rstudio-server \
-B /home/ubuntu/rnaseq23/day2_RStudio/working_dir:/home \
rstudio_4.1.0.wCP.sif \
rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user ubuntu
```

Intermittently receiving this error: 
``` 
FATAL:   container creation failed: mount /proc/self/fd/3->/usr/local/var/singularity/mnt/session/rootfs error: while mounting image /proc/self/fd/3: failed to find loop device: could not attach image file to loop device: no loop devices available
``` 

Current workaround: 

1. Check if the loop module is loaded:

```
lsmod | grep loop
```

2. If the loop module is not listed, you can load it using: 

```
sudo modprobe loop
```

3. Verify the loop devices availability: 

```
sudo losetup -f
```


This issue has been observed intermittently on current bioimage when running Singularity, RStudio, and Nextflow. Contacting Pawsey Helpdesk for support.
